yt_dlp/extractor/twitch.py

```diff
@@ -314,6 +376,33 @@ def _extract_info_gql(info, item_id):
         if thumbnail:
             for p in ('width', 'height'):
                 thumbnail = thumbnail.replace('{%s}' % p, '0')
+
+        chapters = list()
+        for moment in info.get('moments'):
+            moment = moment.get('node')
+            if moment.get('type') != 'GAME_CHANGE':
+                # TODO: warning or debug log if found unknown moment type
+                continue
```

Comment:
why? The type doesnt affect anything in code

Reply:
moments seem to be a multipurpose feature (but for now this might be the only type that exists), and I have a feeling that when Twitch modifies this feature, chapter extraction might silently break and I'll never notice.
probably this is silly, but I see it the way as there is a warning too when a description was not found: it hints that something might be off there.